app/api/chat/openai/route.ts

```diff
@@ -30,7 +30,8 @@ export async function POST(request: Request) {
       temperature: chatSettings.temperature,
       max_tokens:
         chatSettings.model === "gpt-4-vision-preview" ||
-        chatSettings.model === "gpt-4o"
+        chatSettings.model === "gpt-4o" ||
+        chatSettings.model === "gpt-4o-mini"
```

Comment:
mini supports 16k output tokens - would this limit it to 4k?

Reply:
it will.

It should be

```js
const response = await openai.chat.completions.create({
  model: chatSettings.model as ChatCompletionCreateParamsBase["model"],
  messages: messages as ChatCompletionCreateParamsBase["messages"],
  temperature: chatSettings.temperature,
  max_tokens:
    chatSettings.model === "gpt-4-vision-preview" ||
    chatSettings.model === "gpt-4o"
      ? 4096
      : chatSettings.model === "gpt-4o-mini"
      ? 16383
      : null, 
  stream: true
});
```

Reply:
yeah right fixed!! thanks